README.rst

```diff
@@ -25,8 +25,7 @@ Important note
 **************
 
 It's important that this package isn't started by cronjob in non-production
-deployments. This is because the command is started by a zope
-ctl command.
+deployment. See non-production-info_ for more information.
```

Comment:
So the point is that in a ZEO setup where the ZEO server might be down longterm for valid reasons - it could lock up/crash the entire machine.  Non zeo setups (e.g. development) would be fine as bin/instance isn't a ZEO Client.

I think I'd say.....

*Important:* You should be careful not to activate this script as a cronjob in environments where the ZEO server could be legitimately down for long periods (e.g. Staging / Test servers) as it could lead to locking up/crashing the entire machine.

What do you think?

Reply:
Sounds good: Adapted in 6badd5c